eclair-core/src/test/scala/fr/acinq/eclair/transactions/TransactionsSpec.scala

```diff
@@ -38,6 +40,20 @@ import scala.util.{Failure, Random, Success, Try}
 
 class TransactionsSpec extends FunSuite with Logging {
 
+  val localFundingPriv = PrivateKey(randomBytes32 :+ 1.toByte)
```

Comment:
Why the `:+ 1.toByte`?

Reply:
I think it was a leftover from when we used to initialize private keys in compressed form but now it's useless.

Reply:
Gotcha, good to remote it then, if we can make these tests slightly easier to read it's a great win!